.github/workflows/ci.yml

```diff
@@ -31,6 +31,7 @@ env:
 jobs:
   build_test:
     runs-on: ubuntu-latest
+    needs: generate_cache
```

Comment:
Can you add a comment here something to this effect?

> Generate and populate the global Cargo registry and cache first. Each job in the matrix runs in parallel, so without populating the cache first, most jobs would duplicate the work of downloading crates from the internet. Populating the cache first ensures that this work only happens once.

Reply:
Sure